src/xFactor/Judge.java

```diff
@@ -0,0 +1,104 @@
+package xFactor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Judge extends Person {
+	private static int maxFavourites;
+	private static int numberOfJudges;
+	private static int countInstances;
+	private static ArrayList<Judge> judges = new ArrayList<Judge>(numberOfJudges);
+	private int vote;
+	private boolean isSpecial;
+	private ArrayList<Participant> favourites = new ArrayList<Participant>(maxFavourites);

```

Comment:
The arrayList is dynamic sized collection. The benefit compared to the simple array is that you do not need to know the size of the collection to create it